objectRecognitionSource/build.gradle

```diff
@@ -197,7 +197,7 @@ dependencies {
     
     // Used by YoloProcessor alone. These two dependencies and all files in the edu.* package may be removed if YoloProcessor
     // is removed
-    compile "org.tensorflow:tensorflow:1.9.0"
+    compile "org.tensorflow:tensorflow:1.13.1"
```

Comment:
Before we check this in, we should check with Brett and/or Patrick.  TensorFlow versions >= 1.12 require a GPU with a CUDA Count (a measure of the GPU's power) greater than 6.  Some of the machines they use may not have that power (those are represented in "Data Center" class GPUs, with costs much higher than typical PC GPU prices).  So we might want to think carefully here.

1.11.* is still OK with the GPUs of mere mortals